dataactcore/interfaces/dbInterface.py

```diff
@@ -0,0 +1,40 @@
+import uuid
+
+from dataactcore.models.userModel import User, UserStatus
+from dataactcore.interfaces.db import databaseSession
+
+# First step to deprecating BaseInterface, its children, and corresponding
+# interface holders is to start moving all db access logic into one big
+# file (to prevent circular imports and have everything in the same place).
+# Still to do...make this work solely with a flask context...original idea
+# was that these functions would only be invoked within a Flask route, but
+# there are some (e.g., createUserWithPassword) that need to be here,
+# pending a further refactor.
+# As a temporary measure, until the next part of the work that refactors
+# the db access within Flask requests, fire up an ad-hoc db session in
+# these transitional functions.
+
+
+def createUserWithPassword(email, password, bcrypt, permission=1, cgac_code="SYS"):
+    """Convenience function to set up fully-baked user (used for setup/testing only)."""
+    with databaseSession() as sess:
+        status = sess.query(UserStatus).filter(UserStatus.name == 'approved').one()
+        user = User(email=email, user_status=status, permissions=permission,
+                    cgac_code=cgac_code)
```

Comment:
Yes, agreed! I'm going to leave this as part of our future work to re-think our overall strategy to dealing with helper table values.